kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -72,6 +86,9 @@
       resource() {
         return this.examOrLesson === 'lesson' ? this.lesson : this.exam;
       },
+      isFromOldKolibiri() {
+        return this.exam.data_model_version < 3 && this.exam.draft && !this.exam.active;
```

Comment:
I don't see the notice and I believe it's because of the `this.exam.draft` which is only set on things that